main.py

```diff
@@ -153,26 +169,17 @@ def handle_trade(trade: Trade) -> None:
         current_rate = api_wrapper.get_ticker(trade.pair)['bid']
         current_profit = 100 * ((current_rate - trade.open_rate) / trade.open_rate)
 
-        # Get available balance
-        currency = trade.pair.split('_')[1]
-        balance = api_wrapper.get_balance(currency)
+        if ('stoploss' in CONFIG) & (current_profit < float(CONFIG['stoploss'])*100):
```

Comment:
Thanks for the PR

Please write this condition in a more idiomatic way, e.g.:
```
if 'stoploss' in CONFIG and current_profit < float(CONFIG['stoploss']) * 100:
```

cheers

Reply:
Fixed in 	69bfc2d